integration_tests/dbt_project.yml

```diff
@@ -10,4 +10,7 @@ dispatch:
 
 vars:
   aws_cloud_cost:
-    aws_cloud_cost_schema: aws_cloud_cost_integration_tests_3
\ No newline at end of file
+    aws_cloud_cost_schema: aws_cloud_cost_integration_tests_3
+
+models:
+  +schema: "aws_cloud_cost_{{ var('directed_schema','dev') }}"
```

Comment:
I know you are still finishing out BK, but just adding this here to be included.